install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -0,0 +1,273 @@
+[[inventory_provision]]
+=== Preparing the Inventory for Provisioning
+
+With the installation of the `openshift-ansible` package complete via our
+previous steps, there resides a 
+`sample-inventory` directory that we will copy to our `cloud-user` home directory
+of the deployment host.
+
+On the deployment host, 
+
+----
+$ cp -r /usr/share/ansible/openshift-ansible/playbooks/openstack/sample-inventory/ ~/inventory
```

Comment:
The correct command is :  cp -r /usr/share/ansible/openshift-ansible/playbooks/openstack/sample-inventory/### ***** ~/inventory

Reply:
I just reconfirmed. It is not with the *. We want it to copy the full sample-inventory dir and place it into a new dir under ~/inventory